src/store/types.ts

```diff
@@ -37,3 +37,7 @@ export interface ITimeRecorderState {
   tmpState: ITimerState,
   isActive: boolean,
 }
+
+export interface ITagManager {
```

Comment:
```suggestion
export interface ITagManagerState {
```

他のinterfaceは全部最後が `State` で統一しているから、ここも